lib/main.dart

```diff
@@ -39,24 +40,7 @@ class _ParkrAppState extends State<ParkrApp> {
     if (widget.startupException != null) {
       throw widget.startupException as DisplayableException;
     }
-
-    // navigate straight to home page if already signed in
```

Comment:
Does this just alway put you to the login?

Reply:
yes, instead of being sent directly to the homepage when the app loads and you are already signed in, you will have to sign in again. The fetchAuthUserSession was failing badly. Even when internal cognito errors occurred, it still said the user was signed in. I removed it to avoid those issues.

Reply:
okay, just wanted to make sure.